api/hypershift/v1beta1/hosted_controlplane.go

```diff
@@ -302,6 +302,18 @@ type HostedControlPlaneStatus struct {
 	// +optional
 	ReleaseImage string `json:"releaseImage,omitempty"`
 
+	// AvailableUpdateVersions is a comma separated list of all available update versions.
+	//
+	// Deprecated: Use versionStatus.availableUpdates instead.
+	// +optional
+	AvailableUpdateVersions string `json:"availableUpdateVersions,omitempty"`
```

Comment:
New structured properties seems heavy for a throw-away property we want to drop later?  Could we use a custom `type` for `status.conditions[]` to set a string without getting tied up in CRD structure?

Reply:
adding a new field seems a bit more consistent, if you look at the ones above, which are also created for the purposes of the feedback rules

@sjenning any opinions on this?